extensions/aligning_guidelines/util/make.ts

```diff
@@ -0,0 +1,32 @@
+import type { HorizontalLineProps, VerticalLineProps } from '../typedefs';
+import { AligningLineConfig } from '../constant';
+
+export function makeVerticalLine(props: VerticalLineProps) {
+  const aligningLineOffset = AligningLineConfig.offset;
+  const { x, objectHeight, objectY, activeObjectHeight, activeObjectY } = props;
+  const b = objectY < activeObjectY;
+  return {
+    x,
+    y1: b
+      ? objectY - objectHeight / 2 - aligningLineOffset
+      : objectY + objectHeight / 2 + aligningLineOffset,
+    y2: b
+      ? activeObjectY + activeObjectHeight / 2 + aligningLineOffset
+      : activeObjectY - activeObjectHeight / 2 - aligningLineOffset,
+  };
+}
+
+export function makeHorizontalLine(props: HorizontalLineProps) {
+  const aligningLineOffset = AligningLineConfig.offset;
+  const { y, objectX, objectWidth, activeObjectX, activeObjectWidth } = props;
+  const b = objectX < activeObjectX;
+  return {
+    y,
+    x1: b
+      ? objectX - objectWidth / 2 - aligningLineOffset
+      : objectX + objectWidth / 2 + aligningLineOffset,
+    x2: b
+      ? activeObjectX + activeObjectWidth / 2 + aligningLineOffset
+      : activeObjectX - activeObjectWidth / 2 - aligningLineOffset,
+  };
+}
```

Comment:
@zhe-he this file seems completely unused. Why is it here? can i delete it?

Reply:
Yes, this file is not used anywhere, and you can delete it. In the original alignment version, there is a parameter called aligningLineOffset, and I found that its use case is completely ineffective in the new version. Later, I forgot to delete this file, and the offset property in AligningLineConfig should also be removed.

Reply:
https://github.com/fabricjs/fabric.js/pull/10056